roles/rust-foundation-project-director.md

```diff
@@ -0,0 +1,24 @@
+# Role Description: Rust Foundation Project Director
+
+Please refer to [Rust Foundation Director Roles & Responsibilities](TODO) for components of the role description that apply to all Foundation Board Directors.
+
+## Goals
+
+- Represent the needs and interests of the Rust Project at large within the Rust Foundation board
+- Act as a conduit of information between the Rust Project and the Rust Foundation
+
+## Time Budget
+
+- 2-5 hours per week
+
+## Tasks / Activities / Responsibilities
```

Comment:
Same point as above. Let's keep this list short. fill it with things we know won't change (e.g., attending Rust Foundation board meetings, meeting with the Rust Foundation staff on a regular basis), and then end it with a TODO of helping shape what this role is in the future.

Reply:
Here's my attempt at a diff that adds @rylev's suggestion. I changed meeting with Foundation staff to as needed rather than regular. My reasoning was that I don't know that we should expect Project Directors to need a standing meeting with various staff members, but they should have open communication lines that they use when issues come up that need to be discussed.

I also added communicating with the Rust Project as an explicit item. We might not know exactly what this looks like, but they are supposed to represent the Project on the Foundation Board, so we need two way communication between the two.

```suggestion
## Tasks / Activities / Responsibilities

* Attend Rust Foundation board meetings
* Meeting with Rust Foundation staff as needed
* Communicate with the Rust Project. Project directors have leeway in how to do this, but at a minimum it should include discussing with the Leadership Council issues that are relevant to the Project.
```

Reply:
I wish I hadn't forgot to push the diff I had already done, duplicated work, feelsbadman ;_;